robotbase/src/main/java/com/team2357/frc2022/shuffleboard/CargoCountWidget.java

```diff
@@ -0,0 +1,68 @@
+package com.team2357.frc2022.shuffleboard;
```

Comment:
This class will probably go away in favor of CargoNumberWidget, since it will require to less work to implement functional interface in it. However, the actual code and logic for the widget seems good.